Source/WebCore/bindings/js/JSDOMWindowCustom.cpp

```diff
@@ -224,6 +302,26 @@ bool JSDOMWindow::getOwnPropertySlot(JSObject* object, JSGlobalObject* lexicalGl
     }
 #endif
 
+#if ENABLE(WEB_AUTHN)
+#if !ENABLE(USER_MESSAGE_HANDLERS)
+    RefPtr localDOMWindow = dynamicDowncast<LocalDOMWindow>(thisObject->wrapped());
```

Comment:
Why is marking as Unforgeable in the IDL not sufficient? We should really try to avoid modifying this function.